storops/unity/calculator.py

```diff
@@ -295,6 +313,254 @@ def _sp_pct(path, curr, prev):
     return ret
 
 
+@metric_calculator
+def system_delta_ps(path, prev, curr):
+    path = only_one_path(path)
+
+    if all_not_none(prev, curr):
+        prev_counter = prev.by_path(path)
+        curr_counter = curr.by_path(path)
+    else:
+        prev_counter = None
+        curr_counter = None
+
+    if all_not_none(prev_counter, curr_counter):
+        delta = curr_counter.sum_sp_values - prev_counter.sum_sp_values
+        ret = delta / curr.diff_seconds(prev)
+    else:
+        ret = IdValues()
+    return ret
+
+
+@metric_calculator
+def system_total_delta_ps(path, prev, curr):
+    if len(path) != 2:
+        raise ValueError('takes in "reads" and "writes" counter.')
+
+    part1, part2 = path
+    return system_delta_ps(part1, prev, curr) + system_delta_ps(part2, prev,
+                                                                curr)
+
+
+@metric_calculator
+def disk_response_time(path, prev, curr):
+    if len(path) != 5:
+        raise ValueError(
+            'takes in "busyTicks", "sumArrivalQueueLength", "reads", '
+            '"writes" and "coreCount" counter.')
+
+    part1, part2, part3, part4, part5 = path
+    if all_not_none(prev, curr):
+        prev_part1 = prev.by_path(part1)
```

Comment:
Better to give them metric names instead of `partx`.

Reply:
Done, already rename the variables.